### ISSUE TITLE
Hang at `Configuring and enabling network interfaces...` 

### ISSUE THREAD

**Issue by 340515991** [mitchellh/vagrant#8295]
The box  is custom by myself use centos 7

### Vagrant version
Vagrant 1.9.1

### Host operating system
windows10 

### Guest operating system
centos 7 

### Vagrantfile
```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure(2) do |config|
    config.vm.define "vbox1" do | dev |
        dev.vm.box = "vbox1"
	dev.ssh.insert_key = false
        dev.ssh.username = "root"
        dev.ssh.password = "123456"
	dev.vm.network "public_network", ip: "192.168.0.100"
        dev.vm.synced_folder "../wwwroot", "/data/wwwroot/"
        dev.vm.provider "virtualbox" do | vb |
            vb.name = "vbox1"
            vb.gui = false
            vb.memory = "256"
        end
    end
end
```

### Debug output
https://github.com/340515991/vagrant/blob/master/debug.log

### Expected behavior
What should have happened?

### Actual behavior
What actually happened?

### Steps to reproduce
1.execute vagrant up vbox1 
2.hung at Configuring and enabling network interfaces

**Comment by chrisroberts** [comment]
Hi! The debug log shows the machine successfully being transitioned to the `up` state. Does the error only show up on the initial `up` when the machine is created? If so, can you please provide the debug log for that? Thanks!

**Comment by 340515991** [comment]
yeah ...  here is it the

**Comment by 340515991** [comment]
https://github.com/340515991/vagrant/blob/master/debug2.log

**Comment by famod** [comment]
Sounds like #8148. Have you tried reverting to 1.9.0?

**Comment by chrisroberts** [comment]
#8148 should be the fix for this and is included in the 1.9.2 release. If you are still encountering the error please feel free to reopen this issue.

Cheers!
